packages/xchain-cosmos/__e2e__/cosmos-client.integration.ts

```diff
@@ -0,0 +1,55 @@
+import { Network, TxParams, XChainClient } from '@xchainjs/xchain-client'
```

Comment:
[nitpick] Rename `cosmos-client.integration.ts` to `cosmos-client.e2e.ts` or `cosmos-client.test.ts`

Reply:
done